e2e-tests/data-at-rest-encryption/run

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
I think we need to add more tests like we have in PXC e.g.
'restore-to-encrypted-cluster'

and please check this test case https://github.com/percona/percona-xtradb-cluster-operator/blob/main/e2e-tests/demand-backup-encrypted-with-tls/run#L41-L62

and we need to checks that data is really encrypted/unencrypted   like we have in PXC.

@nmarukovich please think about test cases too and discuss it with @egegunes